### PR TITLE
VORTEX-5489 : Export My Places Sub-Folders

### PR DESCRIPTION
**Pull request by brelsftm**
Fixes #

## Proposed Changes

  - Right clicking on a sub-folder brings up correct context menu.
  - Ability to export specific sub-folders to different file types.

### CODE REVIEW

File: open-sphere-plugins/my-places/src/main/java/io/opensphere/myplaces/controllers/RoiContextMenuProvider.java [hunk 1]
```diff
@@ -42,13 +42,16 @@ public RoiContextMenuProvider(Toolbox toolbox)
             MyPlacesDataTypeInfo type = (MyPlacesDataTypeInfo)key.getDataType();
             Placemark kmlPlacemark = type.getKmlPlacemark();
 
-            if (!(kmlPlacemark.getGeometry() instanceof Polygon))
+            if (kmlPlacemark.getGeometry() instanceof Polygon)
+            {
+                PolygonGeometry geom = RegionUtils.createGeometry(kmlPlacemark);
+                menuItems = MantleToolboxUtils.getMantleToolbox(myToolbox).getSelectionHandler().getGeometryMenuItems(geom);
+            }
+            else
```
Comment (bcleveland1022) [inline open-sphere-plugins/my-places/src/main/java/io/opensphere/myplaces/controllers/RoiContextMenuProvider.java]: Do we need this else anymore with your changes?

Comment (brelsftm) [reply open-sphere-plugins/my-places/src/main/java/io/opensphere/myplaces/controllers/RoiContextMenuProvider.java]: Yeah, this is to distinguish the menu items if its a track vs point vs line . The previous !(instance of ) was not catching a few cases , so this just makes it easier

Comment (brelsftm) [reply open-sphere-plugins/my-places/src/main/java/io/opensphere/myplaces/controllers/RoiContextMenuProvider.java]: Its a little unrelated to my ticket, but I thought Id just fix it while I was getting the error testing this ticket

Comment (brelsftm) [reply open-sphere-plugins/my-places/src/main/java/io/opensphere/myplaces/controllers/RoiContextMenuProvider.java]: no we dont , good catch

**Pull request merged**